ios/AppDelegate.m

```diff
@@ -100,4 +105,11 @@ - (void) showUnrecoverableDialogWithTitle:(NSString *)title message:(NSString *)
   }]];
   [self.window.rootViewController presentViewController:alertController animated:YES completion:nil];
 }
+
+#if ENABLE_DEEP_LINK
```

Comment:
I would not wrap the method but just the implementation.

Reply:
Sure